.github/workflows/tags.yml

```diff
@@ -0,0 +1,23 @@
+name: Bump Version Tag
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  tags:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: "0"
+
+      - name: Bump version and push tag
+        uses: anothrNick/github-tag-action@1.64.0
```

Comment:
is this action version being renovated?

Reply:
Yes, it will be! https://github.com/UCL-MIRSG/.github/blob/1922ae4421dfc47090aa5c67309763f07992f81c/.renovaterc.json#L6

Reply:
![image](https://github.com/UCL-MIRSG/.github/assets/15052188/411e1f1b-dde5-4a80-b6c4-f01d5dd3b464)
Already on it! The `README` was clearly out of date